app/src/main/java/app/revanced/manager/ui/component/bundle/BundlePatchesDialog.kt

```diff
@@ -62,44 +63,204 @@ fun BundlePatchesDialog(
             LazyColumnWithScrollbar(
                 modifier = Modifier
                     .fillMaxWidth()
-                    .padding(paddingValues)
-                    .padding(16.dp)
+                    .padding(paddingValues),
+                verticalArrangement = Arrangement.spacedBy(12.dp),
+                contentPadding = PaddingValues(16.dp)
             ) {
-                item {
-                    AnimatedVisibility(visible = informationCardVisible) {
-                        NotificationCard(
-                            icon = Icons.Outlined.Lightbulb,
-                            text = stringResource(R.string.tap_on_patches),
-                            onDismiss = { informationCardVisible = false }
+                state.patchBundleOrNull()?.let { bundle ->
+                    items(bundle.patches.size) { bundleIndex ->
+                        val patch = bundle.patches[bundleIndex]
```

Comment:
There is an overload for `List`s. I don't know why this wasn't being used in the original file. You may need to import it.

```suggestion
                    items(bundle.patches) { patch ->
```

Reply:
There's no such thing, but this is the next best thing:

```kotlin
items(
    count = bundle.patches.size,
    key = { index -> bundle.patches[index] }
) { bundleIndex ->
    val patch = bundle.patches[bundleIndex]
    
    PatchItem(
        patch,
        showAllVersions,
        onExpandVersions = { showAllVersions = !showAllVersions },
        showOptions,
        onExpandOptions = { showOptions = !showOptions }
    )
}

Reply:
There is: https://developer.android.com/reference/kotlin/androidx/compose/foundation/lazy/LazyListScope#(androidx.compose.foundation.lazy.LazyListScope).items(kotlin.collections.List,kotlin.Function1,kotlin.Function1,kotlin.Function2)